app/assets/stylesheets/custom.scss

```diff
@@ -589,3 +589,9 @@ table {
     }
   }
 }
+
+.budget-heading {
+  background: $brand image-url("background-header-budget.jpg"); //pending  add image from zepling
```

Comment:
I don't see the image on the budgets page. The style is being overwritten by this rule in `participation.scss`:

```scss
.expanded {

  &.budget {
    background: $budget;
  }
```